src/script_opt/ZAM/ZOp.h

```diff
@@ -0,0 +1,65 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+// ZAM instruction opcodes and associated information.
+
+#pragma once
+
+namespace zeek::detail {
+
+// Opcodes associated with ZAM instructions.
+typedef enum {
```

Comment:
`typedef enum` is an old C construct that doesn't mean much in C++ anymore. Just name the enum directly like `enum ZAMOpType` and remove the `typedef`.

Same comment below

Reply:
Fixed.